accounts/accounts.go

```diff
@@ -0,0 +1,54 @@
+package accounts
+
+import "errors"
+
+// Account : State of a particular account
+type Account struct {
+	Balance float64
```

Comment:
Yeah, this is me cutting corners. ✂️ 

How does Go feel about operator overloading? First thing that comes to mind is to introduce a `Currency` type with `.dollars` and `.cents` with `+` and `-` overloaded to handle rollovers / unders between the two values.